CODEOWNERS

```diff
@@ -1 +1 @@
-*       @elatt @oleksandr-saienko @dulcardo @alexromanyuk
+*       @elatt @oleksandr-saienko @dulcardo @alexromanyuk @andrius-senulis
```

Comment:
```suggestion
*       @elatt @alexromanyuk @andrius-senulis
```
Dulcardo and Alex are no longer with the company.

Reply:
Thanks, updated